src/runtime/virtcontainers/pkg/agent/protocols/types.pb.go

```diff
@@ -96,10 +96,8 @@ type Interface struct {
 	IPAddresses []*IPAddress `protobuf:"bytes,3,rep,name=IPAddresses,proto3" json:"IPAddresses,omitempty"`
 	Mtu         uint64       `protobuf:"varint,4,opt,name=mtu,proto3" json:"mtu,omitempty"`
 	HwAddr      string       `protobuf:"bytes,5,opt,name=hwAddr,proto3" json:"hwAddr,omitempty"`
-	// pciAddr is the PCI address in the format  "bridgeAddr/deviceAddr".
-	// Here, bridgeAddr is the address at which the bridge is attached on the root bus,
-	// while deviceAddr is the address at which the network device is attached on the bridge.
-	PciAddr string `protobuf:"bytes,6,opt,name=pciAddr,proto3" json:"pciAddr,omitempty"`
+	// PCI path for the device (see the pci::Path (Rust) or types.PciPath (Go) type for format details)
+	PciPath string `protobuf:"bytes,6,opt,name=pciPath,proto3" json:"pciPath,omitempty"`
```

Comment:
I don't expect we'll be hot plugging devices to an already running pod necessarily, but we should make sure we're careful and announce that this will force an API change between the host-runtime and guest-agent. If you have a long running pod, you'll see an old agent communicating with a new runtime.

Reply:
Actually, it doesn't change the protocol.  We're changing the name for clarity, but it's the same string we're passing through.

Well... we will allow a wider range of paths with this series, but all the existing strings passed here are still valid.

(Which kind of illustrates my point about "PCI address" being a terrible name here)